compiler/modules/nback.llvm/llvm.bal

```diff
@@ -125,6 +158,23 @@ public class FunctionDecl {
         self.functionName = functionName;
         self.functionType = functionType;
     }
+
+    function output(Output out) {
```

Comment:
This function has a lot of common code with `header`. Please factor out the common bits into a separate function.

Reply:
Fixed